### PR TITLE
Tctl admin membership ssantiago

### PR DESCRIPTION
**Pull request by DJSanti**
Documents for admin membership commands

### CODE REVIEW

File: docs/tctl/admin/membership/list_db.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+  - operation-guide
+  - tctl
+---
+
+The `tctl admin membership list_db` command lists the cluster items in a targeted membership.
+
+## Modifiers
+
+The modifiers below change the behavior of the command.
```
Comment (aarohib) [inline docs/tctl/admin/membership/list_db.md:14]:
```suggestion
The following modifiers change the behavior of the command.
```

File: docs/tctl/admin/membership/list_gossip.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: tctl admin membership list_gossip
+description: describing ringpop membership items
+tags:
+  - operation-guide
+  - tctl
+---
+
+The `tctl admin membership list_gossip` command lists the ringpop membership items present on the targeted membership.
+
+The modifier below changes the behavior of the command:
```
Comment (aarohib) [inline docs/tctl/admin/membership/list_gossip.md:12]:
```suggestion
The following modifier changes the behavior of the command:
```

File: docs/tctl/admin/membership/list_db.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+id: list_db
+title: tctl admin membership list_db
+description: describing cluster membership items
```
Comment (flossypurse) [inline docs/tctl/admin/membership/list_db.md:4]:
```suggestion
description: How to describe Cluster membership items
```
Bare minimum improvement - still lacking any real richness.

File: docs/tctl/admin/membership/list_db.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+  - operation-guide
+  - tctl
+---
+
+The `tctl admin membership list_db` command lists the cluster items in a targeted membership.
```
Comment (flossypurse) [inline docs/tctl/admin/membership/list_db.md:10]:
```suggestion
The `tctl admin membership list_db` command lists the Cluster items in a targeted membership.
```

File: docs/tctl/admin/membership/list_db.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Modifiers
+
+The following modifiers change the behavior of the command.
+
+#### `--heartbeated_within value`
+
+Filters the list by last heartbeat time.
```
Comment (flossypurse) [inline docs/tctl/admin/membership/list_db.md:18]:
```suggestion
Filters the list by last Heartbeat time.
```

File: docs/tctl/admin/membership/list_gossip.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+id: list_gossip
+title: tctl admin membership list_gossip
+description: describing ringpop membership items
```
Comment (flossypurse) [inline docs/tctl/admin/membership/list_gossip.md:4]:
```suggestion
description: How to describe ringpop membership items
```

File: docs/tctl/admin/membership/list_db.md (hunk 3 above)
Comment (DJSanti) [reply docs/tctl/admin/membership/list_db.md:4]: Truly, I'm working with what I've been given, and it's not much

**Pull request merged**